upgrade/patches/64_custom_field_option_order.sql

```diff
@@ -2,5 +2,5 @@
  * Control the order of custom fields
  *
  */
-ALTER TABLE {{%custom_field_option}} ADD COLUMN cfo_rank int(10) NOT NULL DEFAULT 0;
+ALTER TABLE {{%custom_field_option}} ADD COLUMN cfo_rank INT UNSIGNED NOT NULL DEFAULT 0 AFTER cfo_fld_id;
 ALTER TABLE {{%custom_field}} ADD COLUMN fld_order_by varchar(20) NOT NULL DEFAULT 'cfo_id ASC'
```

Comment:
always add some `AFTER`, `BEFORE` keyword

also, please avoid using useless commit messages like `Clean up code from code review`, such message does not explain purpose or changes of the commit. in other words it's useless later if looking at git logs what changed.